reference/polkadot-hub/assets.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Polkadot Hub Assets
+description: Learn about asset management on Polkadot Hub, including on-chain assets, foreign asset integration, and XCM for cross-chain asset transfers.
+categories: Polkadot Protocol
+---
+
+# Assets on Polkadot Hub
+
+## Introduction
+
+Polkadot Hub is Polkadot's system parachain for issuing and managing on-chain assets. While the relay chain provides security, Polkadot Hub handles asset logic—minting, burning, transfers, and metadata—efficiently and cost-effectively.
+
+Polkadot Hub supports native assets issued on the parachain and foreign assets from other chains, both of which can move seamlessly across the network via XCM.
+
+This guide explains how assets are created, managed, and moved across chains, including key operations, roles, and the differences between native and foreign assets.
+
+## Why Use Polkadot Hub?
+
+The Polkadot relay chain supports only its native token (DOT). Polkadot Hub fills this gap by providing a standardized framework for creating and managing fungible and non-fungible assets. It enables projects to issue tokens, manage supply, and transfer assets across parachains.
```

Comment:
Similar to my comment in the smart contracts intro, I would not start this by saying what the Polkadot Relay chain limitation or its scope.